rb/build.desc

```diff
@@ -158,6 +158,15 @@ ruby_test(name = "ie",
   deps = [ ":ie" ]
 )
 
+ruby_test(name = "ie-w3c",
+  srcs = [
+    "spec/integration/selenium/webdriver/*_spec.rb",
+    "spec/integration/selenium/webdriver/ie/**/*_spec.rb"
+  ],
+  include = ["rb/spec/integration", "build/rb/lib"],
+  deps = [ ":ie" ]
+)
+
```

Comment:
No, that one is a legitimate failure. Jim hasn't implemented the new w3c `/property` endpoint yet and is working on it. (It doesn't fail on OSS IEDriver because I made an equivalent method to get the property)

The error you would see would be something like this:

    Selenium::WebDriver::Driver one element should find by css selector
     Failure/Error: expect(driver.find_element(css: 'div.content').attribute('class')).to eq('content')

     Errno::ENOENT:
       No such file or directory @ rb_sysopen - .../build/rb/lib/selenium/webdriver/atoms/getAttribute.js

You probably have one in your build directory from building firefox tests. If you ran `go clean //rb:ie-test` you should run into this error.